user_guide/common_features.rst

```diff
@@ -49,10 +49,32 @@ prior agreements.
 
 In OroCRM, all of this data is tracked using three types of records: channels, customers, and accounts.
 
+
+Understanding Accounts, Channels and Customers
+----------------------------------------------
+
+For each :ref:`channel created <user-guide-channel-guide-create>`, there is a set of 
+:term:`entities <Entity>` defined. These entities correspond to types of information collected from the corresponding 
+data source.
+
+One of the entities must represent a :term:`customer <Customer>`.
+
+Once a customer record is created, it is assigned to an account. Several accounts may 
+be :ref:`merged <user-guide-accounts-merge>` into one, regardless of the channels.(For example, if you have a B2B 
+customer that represents some client of yours, and then this client is also buying something from your Magento store).
+
+.. note ::
+
+    Customer record settings and the ways to assign them to an account are specified in the course of customization, 
+    subject to the specific client needs. For example, for :ref:`Magento channels <user-guide-magento-channel>`, a 
+    new account is created for each customer record uploaded to OroCRM in the course of synchronization, whereas for 
+    :ref:`B2B channels <user-guide-magento-channel>` account is a mandatory detail that must be specified when creating 
+    a customer.
```

Comment:
Edit: Customer record settings and the ways to assign them to an account are specified in the course of customization, and are subject to a specific client's needs. For example, for :ref:`Magento channels <user-guide-magento-channel>`, a new account is created for each customer record uploaded to OroCRM in the course of synchronization, whereas for :ref:`B2B channels, <user-guide-magento-channel>` an account is a mandatory detail that must be specified when creating a customer.